src/apiCalls.tsx

```diff
@@ -1,17 +1,35 @@
+import { Answers } from './types';
+
 export const getQuestions = async () => {
-  const response = await fetch('http://localhost:3000/data')
+  const response = await fetch('https://dreamhome-mvp.herokuapp.com/api/v1/education')
```

Comment:
this is the new end-point

Reply:
Looks good! we will have to update the post when ready